core/src/main/kotlin/com/alecstrong/sql/psi/core/mysql/MySql.bnf

```diff
@@ -51,6 +51,11 @@ column_constraint ::= [ CONSTRAINT {identifier} ] (
   implements = "com.alecstrong.sql.psi.core.psi.SqlColumnConstraint"
   override = true
 }
+bind_parameter ::= ( '?' [digit] | ':' {identifier} ) {
```

Comment:
we should remove `[digit]` unless they're also supported in mysql/postgres

Reply:
Good call, will remove.

Reply:
you still want the `?` part, so it would just be

`( '?' | ':' {identifier} )`

Reply:
Right, cause it's an override, no an extension, so I have to preserve the existing behavior.

Reply:
yup, perfect